route/build/src/historyFile.f90

```diff
@@ -515,105 +446,48 @@ SUBROUTINE write_flux_rch(this, hVars_local, index_write, ierr, message)
         if(ierr/=0)then; message=trim(message)//trim(cmessage)//' [array_temp]'; return; endif
       end if
 
-      ! write instataneous local runoff in each stream segment (m3/s)
```

Comment:
Below looks like you were able to really simplify the code which is really great to see.